app/room.py

```diff
@@ -6,33 +6,34 @@
 import random 
 
 class Room(Storage):
-	"""docstring for Room"""
 	"""
+		docstring for Room
```

Comment:
This lines seems unnecessary. If its auto generated you can just remove it and maintain the actual docstring alone